dips/dip-196.md

```diff
@@ -0,0 +1,55 @@
+---
+dip: 196
+title: Onchain Network Identity
+authors: Greg Nazario (@gregnazario)
+status: Draft
+type: Informational
+created: 07/06/2021
+last_updated: 07/06/2021
+issue: https://github.com/diem/dip/issues/196
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Summary
+
+Diem nodes need the ability to find other nodes in the network easily.  Additionally, the nodes need the ability to find 
```

Comment:
```suggestion
Dip-6 defines a method for downstream nodes to learn about validator full nodes and gain access to the Diem network.  In turn, validator full nodes would benefit by being able to distinguish known nodes from unknown.
```

Reply:
I don't really buy the value immediately from being able to discover other nodes other than VFNs.